formulaic/transforms/basis_spline.py

```diff
@@ -98,18 +98,21 @@ def basis_spline(  # pylint: disable=dangerous-default-value  # always replaced
     if df is not None and knots is not None:
         raise ValueError("You cannot specify both `df` and `knots`.")
 
+    # Ensure ndarray to avoid conversion overhead
```

Comment:
nit: We can probably do without this comment.

Reply:
Removed.